### PR TITLE
Quick demo fix

### PR DESCRIPTION
**Pull request by Lanre-B**

### CODE REVIEW

File: mlops_ml_models/models_template_notebook.ipynb [hunk 1]
```diff
@@ -2,9 +2,184 @@
  "cells": [
```
Comment (konradbcredera) [inline mlops_ml_models/models_template_notebook.ipynb:2]: Are any of those changes actually needed or it's just output from the notebook run?

Comment (Lanre-B) [reply mlops_ml_models/models_template_notebook.ipynb:2]: Ah, this is just output from the notebook run, this has to do with the first cell.

**Review by konradbcredera — CHANGES_REQUESTED** [review]

**Review by konradbcredera — CHANGES_REQUESTED** [review]
Review done

File: mlops_ml_models/models_template_notebook.ipynb (hunk 1 above)
Comment (Lanre-B) [reply mlops_ml_models/models_template_notebook.ipynb:2]: The outputs will change each time we run the notebook and GitHub will pick it up. It has always been that way. I have not changed much, I only changed the arrangement of the notebook cells to look the way we had it in our first demo cos I did not download the notebook we used in the first demo, so this notebook looks better. And I added the picture of the Architectural diagram to the first cell. So no major changes. 

If I should rerun the notebook and push now, GitHub will assume I changed the cells which is not the case.

Comment (Lanre-B) [reply mlops_ml_models/models_template_notebook.ipynb:2]: Sorry, I misread your message, I will try to find a way to clear it.

**Review by konradbcredera — APPROVED** [review]

**Pull request merged**